tagr/storage/aws_helper.py

```diff
@@ -0,0 +1,84 @@
+'''
+code heavy helper functions for Aws.py
+
+'''
+import boto3
+
+class AwsHelper:
+    def get_matching_s3_objects(self, bucket, object_path='', max_keys_per_request=1):
+        """
+        List objects in an S3 bucket.
+
+        Parameters
+        ----------
+        bucket: Name of the S3 bucket.
+        object_path: Path that leads to all tables
+        max_keys_per_request: number of objects to list down
+        """
+
+        s3 = boto3.client('s3')
+        kwargs = {'Bucket': bucket}
+
+        # If the prefix is a single string (not a tuple of strings), we can
+        # do the filtering directly in the S3 API.
+        if isinstance(object_path, str):
+            kwargs['Prefix'] = object_path
+        else:
+            kwargs['Prefix'] = str(object_path)
+
+        kwargs['MaxKeys'] = max_keys_per_request
+
+        while True:
+
+            # The S3 API response is a large blob of metadata.
```

Comment:
blob or a dict of objects?

Reply:
After debugging, found out its a dict, idk this comment was just a copypasta